lib/tests/streamlit/web/server/server_test.py

```diff
@@ -13,66 +13,60 @@
 # limitations under the License.
 
 """Server.py unit tests"""
+
 import asyncio
 import errno
 import os
 import shutil
 import tempfile
 import unittest
 from unittest import mock
-from unittest.mock import MagicMock, patch
+from unittest.mock import patch
 
 import pytest
 import tornado.httpserver
+import tornado.httpserver
+import tornado.httpserver
 import tornado.testing
+import tornado.testing
+import tornado.testing
+import tornado.web
 import tornado.web
+import tornado.web
+import tornado.websocket
+import tornado.websocket
 import tornado.websocket
 
 import streamlit.web.server.server
-from streamlit import config, RootContainer
-from streamlit.cursor import make_delta_path
-from streamlit.elements import legacy_data_frame as data_frame
-from streamlit.runtime.forward_msg_cache import ForwardMsgCache
-from streamlit.runtime.forward_msg_cache import populate_hash_if_needed
+from streamlit import config
 from streamlit.logger import get_logger
 from streamlit.proto.ForwardMsg_pb2 import ForwardMsg
+from streamlit.runtime.forward_msg_cache import populate_hash_if_needed
 from streamlit.runtime.uploaded_file_manager import UploadedFileRec
 from streamlit.watcher import event_based_path_watcher
-from streamlit.web.server.server import DebugHandler
-from streamlit.web.server.server import HealthHandler
 from streamlit.web.server.server import MAX_PORT_SEARCH_RETRIES
-from streamlit.web.server.server import MessageCacheHandler
 from streamlit.web.server.server import RetriesExceeded
 from streamlit.web.server.server import Server
 from streamlit.web.server.server import State
-from streamlit.web.server.server import StaticFileHandler
 from streamlit.web.server.server import start_listening
-from streamlit.web.server.server_util import (
-    is_cacheable_msg,
-    is_url_from_allowed_origins,
-    serialize_forward_msg,
-)
+from .message_mocks import create_dataframe_msg
 from .server_test_case import ServerTestCase
 
 LOGGER = get_logger(__name__)
 
 
-def _create_dataframe_msg(df, id=1) -> ForwardMsg:
-    msg = ForwardMsg()
-    msg.metadata.delta_path[:] = make_delta_path(RootContainer.SIDEBAR, (), id)
-    data_frame.marshall_data_frame(df, msg.delta.new_element.data_frame)
-    return msg
-
-
 def _create_script_finished_msg(status) -> ForwardMsg:
     msg = ForwardMsg()
     msg.script_finished = status
     return msg
 
 
-class ServerTest(ServerTestCase):
-    _next_session_id = 0
+def _patch_local_sources_watcher():
+    """Return a mock.patch for LocalSourcesWatcher"""
+    return patch("streamlit.web.server.server.LocalSourcesWatcher")
```

Comment:
This patch declaration is in tons of the server tests; I pulled it up here to make things a bit less verbose